rxjava-test/src/main/java/com/elpassion/android/commons/rxjavatest/CompletableExtension.kt

```diff
@@ -0,0 +1,15 @@
+package com.elpassion.android.commons.rxjavatest
+
+import org.mockito.stubbing.OngoingStubbing
+import rx.Completable
+import rx.observers.TestSubscriber
+
+fun OngoingStubbing<Completable>.thenNeverending() = thenReturn(Completable.never())
```

Comment:
I would rename it. it is not only never ending but never sending anything.
name: thenNever would suggest that it just use .never() factory method.